app/src/main/java/app/revanced/manager/ui/screen/settings/update/ChangelogsScreen.kt

```diff
@@ -52,7 +54,12 @@ fun ChangelogsScreen(
         ) {
             if (changelogs == null) {
                 item {
-                    LoadingIndicator()
+                    Box(
+                        modifier = Modifier.fillMaxWidth(),
+                        contentAlignment = Alignment.Center
+                    ) {
+                        LoadingIndicator()
+                    }
```

Comment:
Why was this added? seems random

Reply:
To center it in the middle, otherwise its on the left edge of the screen

Reply:
In that case, can you push it to the compose-dev branch as its own commit? It's weird to have it as part of this PR

Reply:
I changed it because i removed padding from `LoadingIndicator`

Reply:
Not necessarily but it makes `PatcherScreen` actually readable

Reply:
That is the goal of this PR so fair enough.

Would it be possible to create a `CenteredLoadingIndicator` composable instead of creating boxes everywhere?

Reply:
It is a pattern that is being duplicated so it makes sense to make a function. This could even be an extension function since most of these uses are on `LazyColumn`, which has a scope

Reply:
I just set the column containing the `LoadingIndicator` to `CenterHorizontally`

Reply:
That works

Reply:
Yes